synthtool/languages/java.py

```diff
@@ -109,54 +109,41 @@ def fix_grpc_headers(grpc_root: Path, package_name: str) -> None:
     )
 
 
-def gapic_library(
-    service: str,
-    version: str,
-    config_pattern: str = "/google/cloud/{service}/artman_{service}_{version}.yaml",
-    package_pattern: str = "com.google.cloud.{service}.{version}",
-    gapic: gcp.GAPICGenerator = None,
-    **kwargs,
-) -> Path:
-    if gapic is None:
-        gapic = gcp.GAPICGenerator()
+def _common_generation(
+    service: str, version: str, library: Path, package_pattern: str, suffix: str = ""
```

Comment:
Please add doc comments for the 3 functions touched in this PR.

Reply:
Done.